doc/administrator/mapfile.rst

```diff
@@ -8,29 +8,20 @@ administrator manages the application through the database, and the
 application's MapServer mapfile.
 
 The application's mapfile is where WMS and WFS layers are defined.  WMS is used
-for the map (``WMS GetMap``), and for the ``point query`` feature (``WMS
-GetFeatureInfo``). WFS is used for the ``box query`` and ``query builder``
-features (``WFS GetFeature``).
+for the map (``WMS GetMap``), and for the ``point query`` feature (``WMS GetFeatureInfo``).
+WFS is used for the ``query`` (box and point), the ``filters``, and the WFS permalink features.
 
 The application's mapfile is located in the ``mapserver`` directory, it is
 commonly named ``mapserver.map.mako``.
 
 .. note::
 
     The application's mapfile is a ``.mako`` file because it contains variables.
-    These variables are substituted when the ``make`` build command is
-    executed.
+    These variables are substituted when the ``make`` build command is executed.
```

Comment:
is this uptodate regarding docker world?

Reply:
Yes it steel valid :-)